app/views/templates/_related_ministers.html.haml

```diff
@@ -1,5 +1,5 @@
 - if related_ministers.present?
-  %h2 Our ministers
+  %h2= 'Our ' + (related_ministers.count == 1 ? 'minister': 'ministers')
```

Comment:
That's what I thought but the pluralize helper prefixes the string with the number of things ie. "Our 1 minister", "Our 2 ministers". Looking inside the helper implementation, one could use the .pluralize string method but overkill for this case.